test/parallel/test-whatwg-url-properties.js

```diff
@@ -1,7 +1,9 @@
+// Flags: --expose-internals
 'use strict';
 require('../common');
 const assert = require('assert');
 const { URL, URLSearchParams } = require('url');
+const { formatSymbol } = require('internal/url');
```

Comment:
Instead of using `formatSymbol`, you can instead test `url.format` which does the same thing but is a public API.

https://github.com/nodejs/node/blob/3000d770179029094f7a80ac0a0c95f6a00a01c3/lib/url.js#L591

Reply:
Yes, you're right. Updating the pull request.